tests/problems/generic/test_fgw_problem.py

```diff
@@ -27,7 +27,7 @@ class TestFGWProblem:
     @pytest.mark.fast()
     def test_prepare(self, adata_space_rotate: AnnData):
         expected_keys = [("0", "1"), ("1", "2")]
-        problem = FGWProblem(adata=adata_space_rotate)
```

Comment:
When changing default `alpha` to 1 we will have to set `alpha` in this file explicitly in all tests.

Reply:
Done.